tools/jslint.js

```diff
@@ -240,6 +246,11 @@ if (cluster.isMaster) {
     if (files instanceof Array) {
       // Lint some files
       const report = cli.executeOnFiles(files);
+
+      // If we were asked to fix the fixable issues, do so.
+      if (cliOptions.fix)
+        CLIEngine.outputFixes(report);
```

Comment:
No, that appears to be done by passing `fix: true` to the constructor. Much to my surprise, passing that to the constructor is not enough to actually *fix* the fixable issues. That's what this line does.